index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="./assets/css/main.css" />
+  <link rel="stylesheet" href="./assets/css/all.min.css" />
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo</title>
+</head>
+<body>
+  <div class="site-wrapper">
+    <div class="content">
+      <h1 class="title">Todo</h1>
+      <ul class="filter-list">
```

Comment:
you can put this ul inside nav tag. since it's actually a navigation behaviour.

Reply:
Done.